src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped work");
```

Comment:
Similarly to the `doWork` method, the message in `stopWork` should also be more informative. Indicate that it's a Truck that has stopped its work, for example: 'Truck has stopped working.'